### PR TITLE
Add common gitignore

### LINKED ISSUE #36: Copy over common .gitignore
From https://github.com/catalystneuro/.github/blob/main/.gitignore

### PR DESCRIPTION
**Pull request by garrettmflynn**
fix #36 

Do you want this to let the `codecov.xml` file through?

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
> Do you want this to let the codecov.xml file through?

I can't think of a reason to set that to ignore aside from the fact that it's more CI-related

File: codecov.xml
Comment (CodyCBakerPhD) [inline codecov.xml:1]: Wait, this is weird. This used to be called 'coverage.xml'

I think `codecov.xml` was the configuration file for the bot?

Comment (CodyCBakerPhD) [reply codecov.xml:1]: Ah, that's codecov.yml

https://github.com/catalystneuro/neuroconv/blob/main/.github/workflows/testing.yml#L141

What would you think about renaming the output of raw coverage files here to `coverage.xml` (in which case already in our global ignore pattern - and more obvious then, I would have thought 'codecov' was more about the service than the content from the name)

Comment (garrettmflynn) [reply codecov.xml:1]: That's fine. Just pushed

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**